irods_consortium_continuous_integration_test_hook.py

```diff
@@ -12,7 +12,7 @@ def Indexing_PackageName_Regex( package_ext, technology = 'elasticsearch' ):
     tech = re.escape(technology)
     ext = re.escape(package_ext)
     return re.compile(
-        r'irods-rule-engine-plugin-(document-type|{tech}|indexing)-[0-9].*\.{ext}$'.format(**locals())
+        r'irods-rule-engine-plugin-(document-type|{tech}|indexing)_[0-9].*\.{ext}$'.format(**locals())
```

Comment:
4-2-stable seems to have fixed this, so let's go with that solution:
https://github.com/irods/irods_capability_indexing/blob/0a00265cf0a8886c6e55f74f2fafd5c6abdcd2e7/irods_consortium_continuous_integration_test_hook.py#L15

Reply:
Done